Resources/Prototypes/Body/Parts/base.yml

```diff
@@ -10,6 +10,10 @@
     damageContainer: Biological
   - type: BodyPart
   - type: Gibbable
+    gibPrototypes: 
+     - FoodMeatHuman
+     - MaterialBones
```

Comment:
This is a full 30 stack of bones which looks a bit weird, and also there are issues involving other species getting these giblets. Slimes should not drop bones and human meat.